engine/class_modules/sc_death_knight.cpp

```diff
@@ -3222,7 +3220,7 @@ struct apocalypse_t : public death_knight_melee_attack_t
       p() -> pets.apoc_ghouls.spawn( summon_duration, n_wounds );
     }
 
-    if ( p() -> azerite.magus_of_the_dead.enabled() )
```

Comment:
Not 100% sure these ok() calls are the right way to go.  But since the duration is on a spell id separate from the talent id I wanted to make sure there was a reference to the talent for easy clean up if the talent is removed.